presto-docs/src/main/sphinx/security/salesforce.rst

```diff
@@ -0,0 +1,78 @@
+============================
+Salesforce Authentication
+============================
+
+Presto can be configured to enable frontend password authentication over
+HTTPS for clients, such as the CLI, or the JDBC and ODBC drivers. The
```

Comment:
What about the web ui?

Reply:
This is a password authenticater and will just work with the form based authentication in the Web UI.  The other authenticater don't really mention the web UI either.

Reply:
Yup, I've tested this previously, https://prestosql.io/docs/current/admin/web-interface.html#authentication.

Reply:
So maybe we should call this out as well then